content/api/commands/intercept.md

```diff
@@ -1451,21 +1452,26 @@ cy.intercept('/users?_limit=+(3|5)')
 ### Cypress.minimatch
 
 Under the hood, Cypress uses the [minimatch](/api/utilities/minimatch) library
```

Comment:
```suggestion
Under the hood, `cy.intercept` uses the [minimatch](/api/utilities/minimatch) library
```

When I read this, I think every invocation of `Cypress.minimatch` adds `{ matchBase: true }` as an option and not just to `cy.intercept`, even with the above context. Is this clear for others or is it just me?

Reply:
Good point!